config/management/genesis/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "libra-genesis-tool"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+description = "Libra Management is a tool used to manage the configuration of a Libra Node"
```

Comment:
nit: perhaps update this description to be about the genesis tool, as it's the genesis module, no?